src/components/autocomplete/Autocomplete.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
no need to create a separate folder for a component with 1 file only (no separate style files or children components, related only to the component, etc.)

But in case you have a folder - it should be named in PascalCase as well (Autocomplete)